src/dxtbx/format/FormatNXmxED.py

```diff
@@ -63,3 +67,45 @@ def _beam(self, index=None):
         beam.set_polarization_fraction(0.5)
 
         return beam
+
+
+class FormatNXmxEDeBIC(FormatNXmxED):
+
+    """Set a mask specific for the SINGLA installed at eBIC"""
+
+    @staticmethod
+    def understand(image_file):
+        with h5py.File(image_file) as handle:
+            name = dxtbx.nexus.nxmx.h5str(FormatNXmxEDeBIC.get_instrument_name(handle))
+            if name and "ebic" in name.lower():
+                return True
+        return False
+
+    def _start(self):
+        super()._start()
+
+        self._extend_static_mask()
+
+    def _extend_static_mask(self):
+        """Extend the existing static mask to include the flange shadow."""
+
+        pixel_mask = self._static_mask[0]
+        flange_mask = flex.bool(flex.grid(pixel_mask.all()), True)
+
+        # Add mask for shadow of vacuum flange on detector
```

Comment:
Pls add something line 

```
N.B. this is the _inverted_ mask as we will use ~mask below
```

As *documentation* which could be used by developers in the future